tools/docker/docker-compile.pl

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env perl
+
+# copyright: Maciej Pasternacki
+# http://3ofcoins.net/2013/09/22/flat-docker-images/
+#
+# TODO: replace with something homemade
+
```

Comment:
Well, technically he didn't even add his name in, I did that ;)
Should I strip his name out? Or is it polite to leave that in